packages/workflow/src/records/handler/download.ts

```diff
@@ -33,7 +33,8 @@ function getDownloadedOrAssignedExtension(
 ) {
   if (
     inScope(authHeader, ['declare', 'recordsearch']) ||
-    (hasScope(authHeader, 'validate') && status === 'VALIDATED')
+    (hasScope(authHeader, 'validate') &&
+      ['CORRECTION_REQUESTED', 'VALIDATED'].includes(status))
```

Comment:
Following the above comment, the above expression should look something like this, explicit 👍🏼